src/javascript/app/pages/trade/purchase.js

```diff
@@ -49,7 +49,7 @@ const Purchase = (() => {
         const button             = CommonFunctions.getElementById('contract_purchase_button');
 
         const error      = details.error;
-        const show_chart = !error && passthrough.duration <= 10 && passthrough.duration_unit === 't' && /^(risefall|higherlower|asian|touchnotouch)$/.test(sessionStorage.formname);
+        const show_chart = !error && passthrough.duration <= 10 && passthrough.duration_unit === 't' && /^(risefall|higherlower|asian|touchnotouch|reset)$/.test(sessionStorage.formname);
```

Comment:
I think at this point it's easier to say which contract types we don't want to show chart for. probably it's only digits?